utils/src/ooinstall/cli_installer.py

```diff
@@ -261,23 +307,32 @@ def check_hosts_config(oo_cfg):
             if master_lb[0].master or master_lb[0].node:
                 click.echo('The Master load balancer is configured as a master or node. Please correct this.')
                 sys.exit(0)
+            # Check for another host with same connect_to?
         else:
             message = """
-No HAProxy given in config. Either specify one or provide a load balancing solution
-of your choice to balance the master API (port 8443) on all master hosts.
+WARNING: No master load balancer specified in config. If you proceed you will
+need to provide a load balancing solution of your choice to balance the
+API (port 8443) on all master hosts.
```

Comment:
Current implementation asks for one hostname, I've updated the text a bit to clarify it's gotta be the FQDN. It uses that one hostname for:

openshift_master_cluster_hostname=node2.aos.example.com
openshift_master_cluster_public_hostname=node2.aos.example.com

Do we want to do a second prompt for an internal FQDN for load balancing internal traffic? (I could have it default to the first FQDN, just press enter to use that) Or is this best left to advanced install?